tests/test_consume_whitespace.py

```diff
@@ -0,0 +1,39 @@
+"""All Unit tests which focus on how whitespace is handled by the calculator.
+"""
+import pytest
+
+from calc import INTEGER, EOF, PLUS, Calc, CalcError
+
+
+def test_parse_handles_whitespace_correctly():
+    input_text = " 1 + 1 "
+    calc = Calc(text=input_text)
+    assert calc.parse() == 2
+
+
+def test_parse_handles_pre_whitespace_correctly():
+    # These tests are redudent in that the first should handle this case but
```

Comment:
So normally I'm all for parametrizing tests. And I might be convinced because these are pretty verbose.